### PR TITLE
bug 1490727: Recurring payments user registration styling

### PR DESCRIPTION
**Pull request by joshJarr**
Updates the styles for the user registration page. Enforce text max length and add custom text when the user is redirected from payment flow.

**In this PR**
- https://trello.com/c/vrtPOM4e/5-as-a-user-i-can-read-the-payment-terms-so-that-i-can-make-an-informed-decision

**Notes for QA**

- [ ] Ensure user login page does not change site-wide
- [ ] Ensure login page redirects correctly
- [ ] When following recurring payments flow and creating an account there should be text explaining the flow (see screenshot)

**Screenshots**

screenshot 2018-11-05 at 17 32 30
screenshot 2018-11-05 at 17 32 03

### CODE REVIEW

File: kuma/users/jinja2/socialaccount/signup.html [hunk 1]
```diff
@@ -35,9 +37,21 @@ <h1>{{ _('Create your MDN profile to continue') }}</h1>
             </div>
           {% endif %}
 
-          <form class="submission readable-line-length" method="post" action="{{ url('socialaccount_signup') }}">
+          <form class="submission" method="post" action="{{ url('socialaccount_signup') }}">
               {{ honeypot_field()|safe }}
-
+              <!-- {# Custom text for when the user has been redirected through the recurring paymemnt flow #} -->
```
Comment (matthewhall) [inline kuma/users/jinja2/socialaccount/signup.html]: I'd remove the `<!--` and `-->` here as we already have the Django template comment inside.

File: kuma/users/jinja2/socialaccount/signup.html [hunk 2]
```diff
@@ -35,9 +37,21 @@ <h1>{{ _('Create your MDN profile to continue') }}</h1>
...
+              {% if 'next=' in github_connect_url and 'name' in github_connect_url and 'email' in github_connect_url %}
+              <div class="well well-warning">
```
Comment (matthewhall) [inline kuma/users/jinja2/socialaccount/signup.html]: Indentation here. :)

File: kuma/static/styles/components/well.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+$warning-color: $yellow;
+$error-color: $red;
+$well-highlight-width: 8px;
+
+.well {
+    margin: 2em 0;
+    padding: 2em;
+    position: relative;
+
+    &:before {
```
Comment (matthewhall) [inline kuma/static/styles/components/well.scss]: Not sure I understand the purpose for creating a pseudo element here to apply further styles to the element. Couldn't we just add them to the `.well` class itself?

Comment (joshJarr) [reply kuma/static/styles/components/well.scss]: screenshot 2018-11-07 at 13 34 03The before element styles the left border.
Cant use border-left as the top and bottom must be rounded.

Comment (joshJarr) [reply kuma/static/styles/components/well.scss]: Unless there's a super snazzy way to add that with some CSS magic?

**Pull request closed without merging**